src/odemis/dataio/tiff.py

```diff
@@ -1736,6 +1742,141 @@ def open_data(filename):
     return AcquisitionDataTIFF(filename)
 
 
+class DataArrayShadowTIFF(DataArrayShadow):
+    """
+    This class contains information about a DataArray.
+    It has all the useful attributes of a DataArray, but not the actual data.
+    """
+    def __init__(self, tiff_info, *args,**kwargs):
+        """
+        Constructor
+        tiff_info (dictionary): Information about the source tiff file and directory from which
```

Comment:
Add in the documentation that `tiff_info` can also be a list of dict (instead of a dict).